integration_tests/v_and_v_output/.gitignore

```diff
@@ -0,0 +1 @@
+*.csv
```

Comment:
I kind of like having the directory itself tracked in Git even if the contents are not. Is there a reason to avoid this? I could of course achieve the same goal by moving this to the project level gitignore and adding a `.gitkeep` file or similar, if that is more our style.